pom.xml

```diff
@@ -71,12 +71,13 @@
     </issueManagement>
 
     <properties>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.release>8</maven.compiler.release>
```

Comment:
Generating class files which target that release: https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release